### DISCUSSION TITLE
Polestar location

### DISCUSSION THREAD

**Discussion by evm8052** [pypolestar/polestar_api#253] [Ideas]
Is there a possibility to get the location of the car via the APi? This would make automations at home much easier.

**Comment by wouterhardeman** [comment]
Currently it's not possible. As an alternative, I would recommend installing the Home Assistant app on the car itself. That will send its location to Home Assistant.

**Comment by carrel-gr** [comment]
Speaking of the HA app...  I did that a while back and it worked great.  But then in one of the updates, I lost precise location.  The device_tracker entity still seems to detect Home and Away.  But when I put it on a map, the location is super coarse.  This only happens for the HA app device_tracker on my Polestar.  On my phone it works normally.  And the Polestar HA app is hobbled and won't let me get into settings...  Any thoughts/suggestions?

**Comment by dgomes** [comment]
I advise the use of https://play.google.com/store/apps/details?id=de.ixam97.carStatsViewer.carApp&pli=1

https://community.home-assistant.io/t/integration-of-polestar-2/405501/67?u=dgomes
